map_model/src/objects/traffic_signals.rs

```diff
@@ -1,14 +1,22 @@
+use std::collections::{BTreeMap, BTreeSet};
+use std::convert::TryFrom;
+
+use serde::{Deserialize, Serialize};
+
+use abstutil::{deserialize_btreemap, retain_btreeset, serialize_btreemap, Timer};
+use geom::{Distance, Duration, Speed};
+
 use crate::make::traffic_signals::{brute_force, get_possible_policies};
+use crate::objects::traffic_signals::PhaseType::{Adaptive, Fixed};
 use crate::raw::OriginalRoad;
 use crate::{
     osm, CompressedMovementID, DirectedRoadID, Direction, IntersectionID, Map, Movement,
     MovementID, TurnID, TurnPriority, TurnType,
 };
-use abstutil::{deserialize_btreemap, retain_btreeset, serialize_btreemap, Timer};
-use geom::Duration;
-use serde::{Deserialize, Serialize};
-use std::collections::{BTreeMap, BTreeSet};
-use std::convert::TryFrom;
+
+// The pace to use for crosswalk pace in m/s
+// https://en.wikipedia.org/wiki/Preferred_walking_speed
+const CROSSWALK_PACE: Speed = Speed::const_meters_per_second(1.4);
```

Comment:
Interesting, this is slightly over the 3mph in `Scenario::rand_ped_speed`. Maybe the range there should be updated a bit. (No need to make that change, just a remark)